billboard.py

```diff
@@ -43,13 +43,14 @@ class ChartEntry:
         rank: The track's position on the chart, as an int.
     """
 
-    def __init__(self, title, artist, peakPos, lastPos, weeks, rank):
+    def __init__(self, title, artist, peakPos, lastPos, weeks, rank, newEntry=False):
```

Comment:
No need for the default value. This constructor is only used internally, with all the arguments.